Library/Homebrew/test/rubocops/service_spec.rb

```diff
@@ -19,13 +19,14 @@ class Foo < Formula
     RUBY
   end
 
-  it "reports no offenses when a service block only includes custom names" do
+  it "reports no offenses when a service block includes custom names and requires root" do
```

Comment:
Is there a connection between these two things? If not, these should be two separate tests.

Reply:
I guess `require_root` alone should still report an offense, and `name` alone shouldn't.